README.md

```diff
@@ -70,16 +70,15 @@ npm install --save single-spa-angular
 
 Then create `main.single-spa.ts` with the following content:
 ```typescript
-import {platformBrowserDynamic} from '@angular/platform-browser-dynamic';
-import {ApplicationRef} from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { ApplicationRef } from '@angular/core';
 import singleSpaAngular from 'single-spa-angular';
-import mainModule from './main-module.ts';
-import {Router} from '@angular/router';
+import { Router } from '@angular/router';
+import { AppModule } From './app/app.module';
 
 export default singleSpaAngular({
+  bootstrapFunction: () => platformBrowserDynamic().bootstrapModule(AppModule),
```

Comment:
`bootstrapFunction` is statically analyzeable by angular, whereas passing in the platform and module to single-spa-angular is not. Because it's statically analyzeable, dead code can be removed. Also stuff with swapping out platformBrowserDynamic with platformBrowser